nodejs/eks/cluster.ts

```diff
@@ -462,4 +593,16 @@ export class Cluster extends pulumi.ComponentResource {
 
         this.registerOutputs({ kubeconfig: this.kubeconfig });
     }
+
+    createNodeGroup(name: string, args: ClusterNodeGroupOptions): NodeGroupData {
```

Comment:
I think we should have this return a real `NodeGroup` component - I've done that in #49.